source/template-apps.txt

```diff
@@ -15,8 +15,8 @@ Template Apps
 Overview
 --------
 
-You can get up and running with a {+app+} using one of our premade template apps.
-Template Realm apps bring together many of the building blocks available in {+service+}
+You can get up and running with an {+app+} using one of our premade template apps.
+Template App Services apps bring together many of the building blocks available in {+service+}
```

Comment:
"Template App Services apps" reads really oddly to me. Can we leave the "App Services" out here and rely on the {+service+} source constant to provide that context/product name reference at the end of the line?